### PR TITLE
feat(component): connect ng2-contentful-blog and upgrade angular2-rc.4

### PR DESCRIPTION
**Pull request by natali-abyss**

### CODE REVIEW

File: .config/webpack-karma.config.js [hunk 1]
```diff
@@ -13,7 +13,8 @@ module.exports = {
     loaders: [
       {test: /\.ts$/, loader: 'ts', exclude: [/node_modules\/(?!(ng2-.+))/]},
       {test: /\.html$/, loader: 'raw'},
-      {test: /\.styl$/, loader: 'to-string!css?-url!stylus'}
+      {test: /\.styl$/, loader: 'to-string!css?-url!stylus'},
```
Comment (rychkog) [inline .config/webpack-karma.config.js:16]: do we need stylus loader?

File: app/app.component.ts [hunk 2]
```diff
@@ -56,7 +58,8 @@ export class AppComponent implements OnInit {
     this.angulartics2 = angulartics2;
     this.angulartics2GoogleAnalytics = angulartics2GoogleAnalytics;
     this.breadcrumbsService = breadcrumbsService;
-  }
+
```
Comment (rychkog) [inline app/app.component.ts]: redundant new line

File: app/constIdContentType.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (rychkog) [inline app/constIdContentType.json]: rename to contentfulContentTypeIds.json

File: app/index.ts [hunk 4]
```diff
@@ -4,20 +4,21 @@ import {LocationStrategy, HashLocationStrategy} from '@angular/common';
...
 import {TwitterService} from './shared/services/twitter.service';
-import {ContentfulService} from 'ng2-contentful/src/index';
-import DynamicRouteConfigurator from './shared/services/dynamic-route-configurator.service';
-import {RoutesGatewayService} from './shared/services/routes-gateway.service';
 import {Angulartics2} from 'angulartics2/index';
+import {DynamicContentDetailsComponent} from './shared/components/dynamic-content/dynamic-content-details.component';
 import 'rxjs/add/operator/map';
 import 'rxjs/add/operator/toPromise';
-import {BreadcrumbsService} from './shared/components/breadcrumbs/breadcrumbs.service';
+import {appInjector} from 'ng2-contentful-blog/components/contentfulService/app-injector.tool';
```
Comment (rychkog) [inline app/index.ts]: no, no, no :) should be import {appInjector} from 'ng2-contentful-blog'

File: app/index.ts [hunk 5]
```diff
@@ -30,27 +31,23 @@ Ng2ContentfulConfig.config = {
...
-  );
+bootstrap(AppComponent, [
+  ...ROUTER_PROVIDERS,
+  ...HTTP_PROVIDERS,
+  Angulartics2,
+  TwitterService,
+  ContentfulService,
+  Angulartics2GoogleAnalytics,
+  GAPMINDER_PROVIDERS,
+  provide('ComponentUseDefault', {useValue: DynamicContentDetailsComponent}),
```
Comment (rychkog) [inline app/index.ts:42]: `ComponentUseDefault` Name should be changed to more descriptive one

File: app/index.ts [hunk 6]
```diff
@@ -30,27 +31,23 @@ Ng2ContentfulConfig.config = {
...
+  provide('RootComponent', {useValue: AppComponent}),
+  provide('ContentfulConstantId', {useValue: ContentfulConfig}),
```
Comment (rychkog) [inline app/index.ts:44]: `ContentfulConstantId` should be renamed

File: app/index.ts [hunk 7]
```diff
@@ -30,27 +31,23 @@ Ng2ContentfulConfig.config = {
...
+  provide(LocationStrategy, {useClass: HashLocationStrategy}),
+  provide(PLATFORM_DIRECTIVES, {
+    useValue: ContentfulImageDirective, multi: true
+  })
+]).then(
+  (appRef: ComponentRef<any>) => {
+    appInjector(appRef.injector);
```
Comment (rychkog) [inline app/index.ts:51]: Is this still used?

Comment (natali-abyss) [inline app/index.ts:51]: yep, for RoutesGatewayComponent

**Pull request merged**